Makefile

```diff
@@ -1,6 +1,8 @@
 all: build
 .PHONY: all
 
+export GODEBUG := x509ignoreCN=0
```

Comment:
I would like to leave this as distribution-specific and have our upstream free of it

Reply:
The art-team sets it in their builds.

Reply:
The flag is [embedded in OCP base images since v4.6](https://github.com/openshift/images/commit/fbf868763eaefbea0654c27c82cd958147b19942). As a consequence, our code in this repository expects to run under it. Local runs (triggered with make) should run with the same flag.

If we don't add the flag, the unit tests will fail. And this is a good thing.

So either we add the flag in the Makefile, or at runtime in the test files. I think that here in the Makefile is better because it sets it for every test, thus making the tests behave a little more closely to what they'd do in production.